core/services/fluxmonitorv2/orm.go

```diff
@@ -121,7 +122,7 @@ func (o *orm) CreateEthTransaction(
 	idempotencyKey *string,
 ) (err error) {
 
-	_, err = o.txm.CreateTransaction(txmgr.TxRequest{
+	_, err = o.txm.CreateTransaction(context.TODO(), txmgr.TxRequest{
```

Comment:
What about this one?

Reply:
This one sort of felt like scope creep. Tracing the call back up I didn't see any context that I could pass to this call. Even other methods that need a context are using `context.Background()` like [here](https://github.com/smartcontractkit/chainlink/blob/699c7e43f22229c3411b6992fcc28d40236e2822/core/services/fluxmonitorv2/flux_monitor.go#L746). I could convert the Fluxmonitor [chStop](https://github.com/smartcontractkit/chainlink/blob/699c7e43f22229c3411b6992fcc28d40236e2822/core/services/fluxmonitorv2/flux_monitor.go#L84) to `utils.StopChan` and apply the same changes as VRF here if we don't want to leave this as a TODO.

Reply:
Hmmm that's not great...
Well for cases with a `chan struct{}` at least, you can use `utils.Stop` in place as a cast, just to call the method:
```go
ctx, cancel := utils.StopChan(FluxMonitor.ChStop).NewCtx()
```

Reply:
Can we open some follow up stories? or one big one to audit all of the Background and TODO usages?